CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ version directory, and  then changes are introduced.
 
 ### TODO
 
+### Added
+
+- Enabled TTLAfterFinished feature gate. This allows a TTL controller to clean up jobs after they finish execution.
```

Comment:
```suggestion
- Enable TTLAfterFinished feature gate. This allows a TTL controller to clean up jobs after they finish execution.
```

Nitpick, forgot earlier but whole file is written in present for changes.

Also remove the `### TODO` above.